packages/components/src/local/organisms/planning-force/index.tsx

```diff
@@ -31,26 +28,54 @@ export const PlanningForces: React.FC<PropTypes> = ({ assets, selectedAssets, se
     setSelectedAssets([...selectedAssets])
   }
 
+  const getMarkerOption = useCallback((asset: AssetRow, index: number) => {
+    const loc: LatLng = asset.position ? asset.position : latLng([0, 0])
+    const isSelected = selectedAssets.includes(asset.id)
+
+    return {
+      eventHandlers: {
+        click: (): void => {
+          if (interactive) {
+            handleAssetClick(asset.id)
+          }
+        }
+      },
+      key: `asset-icon-${index}`,
+      position: loc,
+      icon: L.divIcon({
+        iconSize: [30, 30],
+        html: getAssetIcon(asset, isSelected, false),
+        className: styles['map-icon']
+      })
+    }
+  }, [])
+
```

Comment:
@lebaphi   - since the above `useCallback` doesn't have any props, it won't get called if `selectedAssets` changes.  Am I mistaken

(this is the first time I've  worked with `useCallback` - so I'm guessing what is happening)

Reply:
ah I just want to memoized the object function, so we don't need to add any props 😄

Reply:
Without `selectedAssets` as a prop for `useCallback` we do not regenerate the icon to show that some assets are selected:

https://user-images.githubusercontent.com/1108513/198992283-2ddb3468-9153-41a7-8292-2b62d1b8d0c3.mp4

By putting `selectedAssets` in for the `useCallback` we do show the pulses for the selected assets:
![screencast](https://i.gyazo.com/71e5d20575b2c13aaf07f54301b624fa.gif)